app/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# syntax = docker/dockerfile:1.2
+
+FROM ghcr.io/eclipse-velocitas/vehicle-app-cpp-sdk:main as builder
```

Comment:
Explain please why do we have module in repo + sdk as registry

Reply:
The SDK provides a base build image with precompiled 3rd party libraries for the target architecture. This is done to reduce app build times.

Reply:
When something needs explanation, it needs documentation as well ;-)